itest/assets_test.go

```diff
@@ -45,6 +46,9 @@ import (
 	"gopkg.in/macaroon.v2"
 )
 
+// PaymentTimeoutSec is the default payment timeout we use in our tests.
+const PaymentTimeoutSec = 6
```

Comment:
nit: I think we should use a `time.Duration` here, and then juse `Seconds()` further down. Then we don't need the unit in the variable name.